www/content/tutorials/getting-started.md

```diff
@@ -7,3 +7,15 @@ weight = 1
 +++
 
 {{< wip >}}
+# Build from Source
+
+Clone from repository
+```shell
+git clone git@github.com:linkdd/tricorder.git
+cd tricorder
+```
+Compile from source (requirement rust installed via rustup):
+```shell
+cargo install --locked --path . 
+```
+make sure `~/.cargo/bin` is on the PATH
```

Comment:
First letter should be capitalized, maybe use `$PATH` instead of PATH to make it explicit that it's an environment variable, and don't forget punctuation.